Cargo.lock

```diff
@@ -1243,9 +1244,9 @@ dependencies = [
 
 [[package]]
 name = "crossbeam-epoch"
-version = "0.9.4"
+version = "0.9.5"
```

Comment:
Came to say we should bump crossbeam-epoch because it's yanked, and you're one step ahead :sunglasses: